testlang/testlang.py

```diff
@@ -338,17 +345,24 @@ def get_standard_exit(self, utxo_id):
         exit_info = self.root_chain.exits(utxo_id)
         return StandardExit(*exit_info)
 
-    def get_balance(self, account):
-        """Queries the balance of an account.
+    def get_balance(self, account, token=NULL_ADDRESS):
+        """Queries ETH or token balance of an account.
 
         Args:
             account (EthereumAccount): Account to query,
+            token (str OR ABIContract OR NULL_ADDRESS):
+                MintableToken contract: its address or ABIContract representation.
 
         Returns:
             int: The account's balance.
         """
-
-        return self.ethtester.chain.head_state.get_balance(account.address)
+        if token == NULL_ADDRESS:
+            return self.ethtester.chain.head_state.get_balance(account.address)
+        if hasattr(token, "balanceOf"):
+            return token.balanceOf(account.address)
+        else:
+            token_contract = conftest.watch_contract(self.ethtester, 'MintableToken', token)
```

Comment:
What's special about `MintableToken` does it not have `balanceOf`?

Reply:
This is misleading. Token can be an address or a wrapped contract.

Reply:
Would love to be able to change that.